cmakelib/bindings/stack.go

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2019 The Kythe Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package bindings
+
+type varStack struct {
```

Comment:
Is there some reason why this type is separate from `Mapping`? In other words, could you move the fields of `varStack` there and simplify some of the methods?

Reply:
Not really, no. I'll go ahead and merge them.

Reply:
And, actually, I think it makes the most sense to just make it a slice-of-maps as a real stack anyway